static/app/views/releases/detail/overview/releaseComparisonChart/index.tsx

```diff
@@ -111,16 +213,13 @@ function ReleaseComparisonChart({
 
   const releaseSessionsCount = getCount(releaseSessions?.groups, SessionField.SESSIONS);
   const allSessionsCount = getCount(allSessions?.groups, SessionField.SESSIONS);
-  const diffSessionsCount =
-    defined(releaseSessions) && defined(allSessions)
-      ? percent(releaseSessionsCount - allSessionsCount, allSessionsCount)
-      : null;
 
   const releaseUsersCount = getCount(releaseSessions?.groups, SessionField.USERS);
   const allUsersCount = getCount(allSessions?.groups, SessionField.USERS);
-  const diffUsersCount =
-    defined(releaseUsersCount) && defined(allUsersCount)
-      ? percent(releaseUsersCount - allUsersCount, allUsersCount)
+
+  const diffFailure =
+    eventsTotals?.releaseFailureRate && eventsTotals?.allFailureRate
+      ? eventsTotals?.releaseFailureRate - eventsTotals?.allFailureRate
```

Comment:
```suggestion
      ? eventsTotals.releaseFailureRate - eventsTotals.allFailureRate
```

Reply:
Nit: I think ts may complain, so maybe you will have to create a const, but that is a minor thing